src/public/css/styles.css

```diff
@@ -1,3 +0,0 @@
-body{
```

Comment:
Why did you delete these files?

Reply:
The css file is in all commits, but i dont know here